ui/lib/pki/app/components/pki-key-usage.js

```diff
@@ -1 +0,0 @@
-export { default } from 'pki/components/pki-key-usage';
```

Comment:
I used the cli command `ember g component pki-key-usage -in-repo pki`. this command always adds an additional app folder that we don't use in our engine setup. I missed committing this directory when I created this component.